pyproject.toml

```
@@ -17,7 +17,7 @@ classifiers = [
 	"Programming Language :: Python :: 3 :: Only",
 ]
 requires-python = ">=3.8"
-dependencies = []
+dependencies = ["packaging"]  # setuptools dependency
```

Let's make the dependencies multi-line, for easier diffs.

Also, I'm unsure about the comment. It communicates that it is a setuptools dependency currently, but that could change. How about instead a comment indicating that setuptools needs to provide all dependencies?

```suggestion
dependencies = [
	# Setuptools must require these
	"packaging",
]
```

Adding this comment pushes me a little closer to releasing distutils as its own package (with a caveat that it's not meant to be installed anywhere), and then including it as a vendored package in Setuptools (just like any other, with its dependencies). I'm not prepared to do that yet, but it's more tempting now that distutils has dependencies.